kasa/modules/module.py

```diff
@@ -72,11 +72,19 @@ def is_supported(self) -> bool:
 
         return "err_code" not in self.data
 
-    def call(self, method, params=None):
+    def call(
+        self,
+        method,
+        params={},  # noqa: B006 - Kasa Cam devices expect {} instead of None for empty args, all others accept it as well
+    ):
         """Call the given method with the given parameters."""
         return self._device._query_helper(self._module, method, params)
```

Comment:
It is better to keep this (and the one below) as they were, and check for `None`ness inside `_query_helper` where needed.

Reply:
fixed